app/views/cart/show.html.erb

```diff
@@ -29,10 +29,14 @@
         <p><%=number_to_currency(item.price)%></p>
       </td>
       <td>
-        <center>
-          <p><%=quantity%></p>
+        <span id="cart-item-<%=item.id%>-quantity">
+          <p>
+            <%= link_to '-', "/cart?increment_decrement=decrement&item_id=#{item.id}", method: :patch %>
+            <%= quantity %>
+            <%= link_to '+', "/cart?increment_decrement=increment&item_id=#{item.id}", method: :patch %>
+          </p>
           <p><%= link_to "Remove", "/cart/#{item.id}",method: :delete %></p>
-        </center>
+        </span>
```

Comment:
you only use span's as an inline targeting element and it shouldnt be used as a container